mteb/tasks/Classification/guj/GujaratiNewsClassification.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class GujaratiNewsClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="GujaratiNewsClassification",
+        dataset={
+            "path": "Sakshamrzt/IndicNLP-Gujarati",
+            "revision": "b98dbe0d93464c1a4789db07e0702677239c770a",
+        },
+        description="A News classification dataset in Gujarati.",
+        reference="https://github.com/AI4Bharat/indicnlp_corpus/blob/master/ai4bharat-indicnlp-corpus-2020.pdf",
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["guj-Gujr"],
```

Comment:
This would have to be rephrased to use a mapping: 
{"language_on_hf": ["guj-Gujr"], ...} 

When using a multilingual dataset.